extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/NotFoundExceptionMapper.java

```diff
@@ -39,16 +52,21 @@
 @Priority(Priorities.USER + 1)
 public class NotFoundExceptionMapper implements ExceptionMapper<NotFoundException> {
 
+    protected static final String META_INF_RESOURCES_SLASH = "META-INF/resources/";
```

Comment:
Perhaps avoid the `META-INF/resources` duplication?

Reply:
Does it really matter? The JVM inlines them so it does not add any overhead.

Reply:
Of course it doesn't matter :). 
Just as a hygiene thing, nothing more. I'm definitely not going to hold up the PR on this :)